test/unittest/dds/topic/DDSSQLFilter/DDSSQLFilterTests.cpp

```diff
@@ -70,6 +70,48 @@ static const std::map<std::string, std::set<std::string>> type_compatibility_mat
     {"ENUM2_STR", {"ENUM2", "CHAR", "STRING"}}
 };
 
+static const std::vector<std::string> operators
```

Comment:
Is static required here?
Also, I would encapsulate those elements that are only for test propose in `test` namespace

Reply:
Refactor for other PR